### PR TITLE
Add a submitted version of "Project with deleted protocols"

### PR DESCRIPTION
**Pull request by lennym**
This makes it visible to ASRU users so we can test the protocol summary table

### CODE REVIEW

File: seeds/data/project-versions.json [hunk 1]
```diff
@@ -392,7 +392,61 @@
...
+              "maximum-animals": "100",
+              "maximum-times-used": "100"
+            }
+          ]
+        },
+        {
+          "id": "997659c6-6789-49db-a5cd-1cfcb6856406",
+          "steps": [],
+          "title": "Deleted protocol",
+          "deleted": false,
```
Comment (wheelsandcogs) [inline seeds/data/project-versions.json:428]: you've got `deleted` twice?

File: seeds/data/project-versions.json [hunk 2]
```diff
@@ -392,7 +392,61 @@
...
+              "name": "Rats",
+              "reuse": false,
+              "value": "rats",
+              "life-stages": [ "adult" ],
+              "continued-use": false,
+              "maximum-animals": "100",
+              "maximum-times-used": "100"
+            }
+          ],
+          "deleted": true
```
Comment (wheelsandcogs) [inline seeds/data/project-versions.json:443]: I assume this is the correct one

Comment (joefitter) [reply seeds/data/project-versions.json:443]: or the other one

File: seeds/data/project-versions.json (hunk 1 above)
Comment (lennym) [reply seeds/data/project-versions.json]: I just copied the version that was already there. I've fixed both.

**Pull request merged**